Makefile

```diff
@@ -60,11 +60,27 @@ build: GOFLAGS += -i -o cockroach
 build: BUILDMODE = build
 build: install
 
+.PHONY: checkgopath
+checkgopath:
+	@GOPATH_ABS=$$(readlink -f "$$GOPATH"); \
```

Comment:
In general, `$GOPATH` doesn't have to be a single directory. (I know that we've had scripts in the past that assumed it's a single directory; I'm not sure if we still do). If this is meant to enforce the one-directory requirement then the message two lines below should say so; if not we should split on `:` and ensure that we are in a subdirectory of the *first* `GOPATH` entry.